### PR TITLE
Remove env variable from sig-compute lane

### PR DESCRIPTION
**Pull request by vatsalparekh**
Remove env variable from sig-compute lane here because they're being moved [#5464](https://github.com/kubevirt/kubevirt/pull/5464)

### CODE REVIEW

**Review by fgimenez — APPROVED** [review]
/lgtm
/approve
/hold

Thanks! Feel free to unhold when https://github.com/kubevirt/kubevirt/pull/5464 is merged.

**Review by oshoval — COMMENTED** [review]
There are some files on project-infra that still have `owner:@sig-compute`
For example robots/cmd/flake-issue-creator/cluster_failure_test.go (and some more),
in case you want to replace it please with `[sig-compute]` as done the in ref PR

**Comment by fgimenez** [comment]
> There are some files on project-infra that still have `owner:@sig-compute`

Indeed @vatsalparekh could you please replace the label with the new format in all these files? once that is done feel free to unhold, https://github.com/kubevirt/kubevirt/pull/5464 is already merged.

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml [hunk 1]
```diff
@@ -481,7 +479,7 @@ periodics:
           - name: TARGET
             value: "k8s-1.20"
           - name: KUBEVIRT_E2E_SKIP
-            value: "SRIOV|GPU|Operator|\\[sig-network\\]|\\[sig-storage\\]|owner:@sig-compute"
+            value: "SRIOV|GPU|Operator|\\[sig-network\\]|\\[sig-storage\\]|sig-compute"
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:482]: Better if we can please keep one style `\\[sig-compute\\]`

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ presubmits:
             - name: TARGET
               value: k8s-1.20
             - name: KUBEVIRT_E2E_SKIP
-              value: "SRIOV|GPU|Operator|\\[sig-storage\\]|\\[sig-network\\]|owner:@sig-compute"
+              value: "SRIOV|GPU|Operator|\\[sig-storage\\]|\\[sig-network\\]|sig-compute"
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:33]: ditto

**Review by oshoval — COMMENTED** [review]
Thanks, see inline please

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml (hunk 1 above)
Comment (vatsalparekh) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml]: Changed

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml (hunk 2 above)
Comment (vatsalparekh) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml]: Changed

**Review by oshoval — APPROVED** [review]
Thanks

**Pull request merged**